packages/camera/camera_android_camerax/lib/src/camera_selector.dart

```diff
@@ -44,11 +44,20 @@ class CameraSelector extends JavaObject {
   late final CameraSelectorHostApiImpl _api;
 
   /// ID for front facing lens.
+  ///
+  /// See https://developer.android.com/reference/androidx/camera/core/CameraSelector#LENS_FACING_FRONT().
   static const int LENS_FACING_FRONT = 0;
 
   /// ID for back facing lens.
+  ///
+  /// See https://developer.android.com/reference/androidx/camera/core/CameraSelector#LENS_FACING_BACK().
   static const int LENS_FACING_BACK = 1;
 
+  /// ID for external lens.
+  ///
+  /// See https://developer.android.com/reference/androidx/camera/core/CameraSelector#LENS_FACING_EXTERNAL().
+  static const int EXTERNAL = 2;
```

Comment:
No action needs to be taken in this PR, but these could be changed to lowerCamelCase. Since you link the Java docs, it should be fine to name enums following a Dart style.